.github/workflows/docker_ci_pipeline.yml

```diff
@@ -0,0 +1,37 @@
+name: Docker CI pipeline
+
+on:
+  push:
+    branches: [ main ]
+    paths: 'docker/**'
+  pull_request:
+    branches: [ main ]
+    paths: 'docker/**'
+  schedule:
+    - cron: '0 0 * * 0'  # Weekly on Sunday
+
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ros_distro: [humble, rolling]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: docker/images/${{ matrix.ros_distro }}/Dockerfile
```

Comment:
@nahueespinosa shouldn't we specify the [platforms](https://docs.docker.com/build/ci/github-actions/multi-platform/) we want to build for? If not, why install QEMU in the first place? Also, if we are targeting multiple platforms, shouldn't we target multiple platforms for CI? I think we should, but I sense it is out of scope for this PR.

Reply:
I added the specified `linux/amd64` in https://github.com/Ekumen-OS/beluga/pull/202/commits/bca95e21d030e5498325675c396c982d0eae52b2.

> Also, if we are targeting multiple platforms, shouldn't we target multiple platforms for CI?

I agree that it is out of scope. In that case, we may want to run tests on different platforms, not just create a docker image.